wlroots-sys/build.rs

```diff
@@ -192,24 +192,24 @@ fn generate_protocols() {
             "gamma_control"
         ),
         ("./wlroots/protocol/wlr-screencopy-unstable-v1.xml", "screencopy"),
-        ("./wlroots/protocol/screenshooter.xml", "screenshooter"),
         ("./wlroots/protocol/idle.xml", "idle")
     ];
 
     for protocol in protocols {
-        wayland_scanner::generate_c_code(
+        wayland_scanner::generate_code(
             protocol.0,
             output_dir.join(format!("{}_server_api.rs", protocol.1)),
             wayland_scanner::Side::Server
         );
-        wayland_scanner::generate_c_code(
+        wayland_scanner::generate_code(
             protocol.0,
             output_dir.join(format!("{}_client_api.rs", protocol.1)),
             wayland_scanner::Side::Client
         );
-        wayland_scanner::generate_c_interfaces(
+        wayland_scanner::generate_code(
             protocol.0,
-            output_dir.join(format!("{}_interfaces.rs", protocol.1))
+            output_dir.join(format!("{}_interfaces.rs", protocol.1)),
+            wayland_scanner::Side::Server
         );
```

Comment:
Sorry for reacting only now, but with the new version of `wayland-scanner` you don't actually need the `*_interfaces.rs` files any longer. So you can remove this part of the build script as well as the `c_interfaces` module in your library, they are no longer used.

I've now updated the documentation for wayland-scanner, it is not yet published, but is available here: https://smithay.github.io/wayland-rs/wayland_scanner/

Reply:
@vberger Thanks for clearing it up! Very appreciated :+1: